.github/workflows/codeforafrica-deploy-prod.yml

```
@@ -74,12 +76,15 @@ jobs:
         uses: docker/build-push-action@v3
         with:
           build-args: |
-            APP=${{ env.APP }}
-            GHOST_API_KEY=${{ secrets.GHOST_API_KEY }}
-            GOOGLE_API_KEY=${{ secrets.GOOGLE_API_KEY }}
+            MONGODB_URL=${{ secrets.CODEFORAFRICA_MONGO_URL }}/${{ env.APP_NAME }}
```

```suggestion
            MONGODB_URL=${{ secrets.CODEFORAFRICA_MONGODB_URL }}/${{ env.APP_NAME }}
```
Lets match the env var names